src/main/java/org/mockito/internal/stubbing/answers/AnswerFunctionalInterfaces.java

```diff
@@ -0,0 +1,326 @@
+package org.mockito.internal.stubbing.answers;
+
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+/**
+ * Functional interfaces to make it easy to implement answers in Java 8
+ */
+public class AnswerFunctionalInterfaces {
+    /**
```

Comment:
Done.

In a parallel universe, there's a mockito2 with _slightly_ higher coverage :p